### PR TITLE
add landspace plot to the landing page

### PR DESCRIPTION
**Pull request by onursumer**
Latest deploy can be tested here: https://germopath.herokuapp.com/

### CODE REVIEW

File: src/util/PlotUtils.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    ITumorTypeFrequencySummary
+} from "../model/GeneFrequencySummary";
+
+export interface IScatterPlotDatum {
+    x: string;
+    y: string;
+    datum: ITumorTypeFrequencySummary;
+}
+
+// TODO this should be applied to the gene, not to a single data point
```
Comment (onursumer) [inline src/util/PlotUtils.ts:17]: This is not used right now. Should probably remove it.

File: src/components/LandscapePlot.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+    @computed
+    public get genesWithSignificantPathogenicGermlineRatio()
+    {
+        return this.props.frequencyStore.geneFrequencySummaryData.filter(d => {
+            const pathogenicGermline = findPathogenicGermlineFrequency(d);
+            const percentBiallelic = findPercentBiallelic(d);
+
+            return (
+                pathogenicGermline && pathogenicGermline.frequency >= 0.0005 &&
```
Comment (onursumer) [inline src/components/LandscapePlot.tsx:57]: Not sure `0.0005` is the best threshold to filter insignificant genes. Also, we should probably move this to either a constant variable or a component property.

**Comment by onursumer** [comment]
We should probably add a title to the scatter plot. The caption of the figure in the paper is `Germline pathogenicity in prospectively characterized advanced cancers`.

File: src/components/LandscapePlot.tsx (hunk 2 above)
Comment (inodb) [reply src/components/LandscapePlot.tsx:57]: yeah this works for now i think 👍 . If it's more a matter of trying to keep the plot decently small we could also do `Top x` or something

**Review by inodb — APPROVED** [review]
LGTM

**Comment by onursumer** [comment]
These changes are now included in #67, closing.

**Pull request closed without merging**